packages/graph/hash_graph/lib/graph/src/types/schema/property_type.rs

```diff
@@ -1,90 +1,39 @@
-use std::collections::{HashMap, HashSet};
+use std::collections::HashSet;
 
 use serde::{Deserialize, Serialize};
 
 use crate::types::schema::{
-    Array, DataTypeReference, OneOf, PropertyTypeReference, Uri, ValidationError, ValueOrArray,
+    array::TypedArray, data_type::DataTypeReference, object::Object, OneOf, Uri, ValueOrArray,
 };
 
-#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
-#[serde(rename_all = "camelCase")]
-enum ObjectTypeTag {
-    Object,
+/// Property Object values must be defined through references to the same valid URI to a Property
+/// Type.
```

Comment:
I'm not sure what this doc is actually saying. I think this struct is just on about representing a reference to a valid property type URI right?

Reply:
This was a left-over from importing the JSONschema. I removed it